pkg/sidecars/models/interfaces.go

```diff
@@ -1,14 +1,27 @@
 package models
 
-import "context"
+import (
+	"context"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+)
+
+// Consumer interface
+type Consumer interface {
```

Comment:
Despite being in the folder "models" (which will be a "general" file for other broker-sidecars too, right?), this structure is all about Kafka. Might be a good idea to rename it to `KafkaConsumer` or something like that.